networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/libp2p/PeerManagerTest.java

```diff
@@ -91,8 +95,10 @@ public void shouldCreatePeerTypeMetrics() {
         peerId -> 0.0);
 
     for (PeerClientType type : PeerClientType.values()) {
-      verify(gauge).labels(any(), eq(type.getDisplayName()));
+      verify(peerClientLabelledGauge).labels(any(), eq(type.getDisplayName()));
     }
+    verify(peerDirectionLabelledGauge).labels(any(), eq("inbound"));
+    verify(peerDirectionLabelledGauge).labels(any(), eq("outbound"));
```

Comment:
this shows a label was defined, but the large testing gap is any logic around when we're adding/removing from the count.